### DISCUSSION TITLE
Package Installation Failure for Custom Logger & Bypass triggers

### DISCUSSION THREAD

**Discussion by jayeshtejwani** [jamessimone/apex-rollup#297] [Q&A]
Hi,

I don't know if it's only me but facing issues in installing the custom logger package to my sandbox. I have attached the screenshot for reference.

![Screen Shot 2022-04-19 at 2 27 03 PM](https://user-images.githubusercontent.com/18471403/164071075-44546f8b-41b9-46f7-a1aa-d8e1298f10cd.png)

Secondly, I wanted to ask, in our org, we have to bypass triggers during updates of rollups, so is it fine if I edit the class and add our custom code to bypass triggers. I have found the place where the update happens so I would like to insert my code here?

<img width="1025" alt="Screen Shot 2022-04-19 at 2 30 23 PM" src="https://user-images.githubusercontent.com/18471403/164071559-89948c03-abad-41a5-9aca-45fd77baa08a.png">

**Comment by jamessimone** [comment]
@jayeshtejwani I will check on that install failure for the custom logger, there are quite a few people already using it and I haven't heard of issues there. Have you verified that the minimum Apex Rollup version is installed? (`v1.4.7`). As far as where to override the update statements ... that's fine as per your screenshot, but I probably need to build a bridge package so that you can actually upgrade Apex Rollup in the future without having to re-do that change over and over again.

**Reply by jamessimone** [reply]
@jayeshtejwani I was unable to reproduce with the latest versions of Apex Rollup and the Custom Object Logger packages. I would recommend that you ensure your versions are up-to-date when installing the packages!

**Reply by jayeshtejwani** [reply]
@jamessimone 

Thank you for confirming about the update statements for bypassing trigger.

Secondly, I checked the version and I do have the v1.4.7 so I dont know whats wrong with the installation. I will look into how to override the class to help store the logs in our custom object.

<img width="1078" alt="Screen Shot 2022-04-19 at 8 42 13 PM" src="https://user-images.githubusercontent.com/18471403/164124125-45cc47d9-8cbf-4fd5-9471-e6936c236c36.png">

**Reply by jamessimone** [reply]
@jayeshtejwani in the upcoming release, I will provide a mechanism for you to stipulate how to perform updates without having to update `RollupSObjectUpdater` yourself (which will prove a hassle for you in terms of upgrading further down the road): https://github.com/jamessimone/apex-rollup/commit/f3e0d3aaef4fe3590f4b6fbb3c8f3d2a390b64a6
